submissions/mayowaShoyinka/domManipulation/signupValidation.js

```diff
@@ -0,0 +1,66 @@
+let signupForm = document.getElementById('signupValidation'),
+    firstName = document.getElementById('firstName'),
+    lastName = document.getElementById('lastName'),
+    phoneNumber = document.getElementById('phoneNumber'),
+    emailAddress = document.getElementById('emailAddress'),
+    password = document.getElementById('password'),
+    confirmPassword = document.getElementById('confirmPassword');
+
+  function validateSignup(field,regdex,errorOne,errorTwo, event) {
```

Comment:
Well done. There is nothing like regdex, confirm and make sure you got the correct spelling.